deidentification/anonymizer.py

```diff
@@ -382,13 +382,15 @@ def _get_cleaned_value(data_element):
     Gets a cleaned value of data_element value according to its representation.
     """
     if data_element.VR == 'UI':
-        return _generate_uuid(data_element.value)
+        return _generate_dicom_uid(data_element.value)
     if data_element.VR == 'DA':
         return "19700101"
     if data_element.VR == 'TM':
         return "000000.00"
     if data_element.VR == 'DT':
         return "19700101000000.00"
+    if data_element.VR in ['DS', 'FL', 'FD', 'IS', 'SL', 'SS', 'UL', 'US']:
+        return '0'
     return "no value"
 
 
```

Comment:
Maybe you can use` if`, `elif` and `else` statements as you've done in `_apply_action` function. But that's ok with a default value.

Reply:
As we have return statements for each if, I was not forced to use elif. Looking quickly some posts on stackoverflow, it may not be an obvious way to do it, but it may be more readable to use `if`/`elif` instead of successive `if`.